### PR TITLE
test Add reviewdog

### PR DESCRIPTION
**Pull request by xiaoliu10**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  golangci-lint:
+    name: runner / golangci-lint
```
Comment (kezhenxu94) [inline .github/workflows/reviewdog.yml:5]: The name is unnecessarily long, rename to `Review`?

Comment (xiaoliu10) [reply .github/workflows/reviewdog.yml:5]: ok, i will

File: .github/workflows/reviewdog.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: reviewdog
 on: [pull_request]
 jobs:
   golangci-lint:
-    name: runner / golangci-lint
+    name: runner / review
```
Comment (kezhenxu94) [inline .github/workflows/reviewdog.yml:5]: Why keep using "runner", I don't think it makes much sense

File: .github/workflows/reviewdog.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  golangci-lint:
+    name: runner / review
```
Comment (kezhenxu94) [inline .github/workflows/reviewdog.yml:5]:
```suggestion
    name: review
```

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**